package.json

```diff
@@ -1364,6 +1374,11 @@
         {
           "command": "gistpad.renameRepositoryFile",
           "when": "resourceScheme == repo"
+        },
+        {
+          "command": "gistpad.syncGist",
+          "when": "resourceScheme == gist && !gistpad.autoSyncWhenSave",
```

Comment:
Ideally, we'd only show this command when there are actually unsynced changes on the active file. But I'm not sure it's worth bothering with that for now. So this seems cool.